azure-pipelines.yml

```diff
@@ -14,7 +14,10 @@ jobs:
   parameters:
     agentOs: macOs
     beforeBuild:
-    - script: brew install gcc6
+    # XCode 11 doesn't play nicely with gcc workaround by using XCode 10. We can try updating to a newer XCode in the future
+    # Once the problem has been resolved
+    - script: sudo xcode-select --switch /Applications/Xcode_10.3.app
+    - script: brew install gcc
```

Comment:
Is there any particular reason we're using gcc on macOS instead of clang?

Reply:
This was derived from other repos building on macOS. We could try clang out, but I'm not interested in doing it in this PR.